lxd/db/query/transaction.go

```diff
@@ -37,6 +37,33 @@ func Transaction(db *sql.DB, f func(*sql.Tx) error) error {
 	return err
 }
 
+// TransactionCtx executes the given function within a database transaction.
```

Comment:
Please can you add a comment here explaining that it adds a 10s timeout to the transaction run time.